src/android/FirebasePluginMessagingService.java

```diff
@@ -184,8 +185,25 @@ private void sendNotification(String id, String title, String messageBody, Map<S
 
             // Since android Oreo notification channel is needed.
             if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.O) {
-                NotificationChannel channel = new NotificationChannel(channelId, channelName, NotificationManager.IMPORTANCE_HIGH);
-                notificationManager.createNotificationChannel(channel);
+                List<NotificationChannel> channels = notificationManager.getNotificationChannels();
```

Comment:
Just being fussy, but we don't need to check if the channel exists, (Android will only add or update it if something different to what's already there)

Reply:
Not my change, mert. Just merging in from upstream.

Can make changes in our fork if you wanna, but I'd rather keep them in sync if it's no big deal?

Reply:
Nah didn't realize it was from upsteam!!